### DISCUSSION TITLE
Include a Linux binary with each vcpkg tool release

### DISCUSSION THREAD

**Discussion by coryan** [microsoft/vcpkg#19148] [Ideas]
**Is your feature request related to a problem? Please describe.**

When using vcpkg in my CI builds a good amount of time is consumed bootstrapping the vcpkg tool. This can be cached, that it is tedious to support the cache.

When explaining to people (e.g. consumers of our libraries) how to use vcpkg we have to include instructions (or link to instructions) on how to compile the tool.

**Proposed solution**

It would be nice if each vcpkg tool release included a Linux binary to download, like the existing Windows and macOS binaries. To minimize problems, this binary can be statically linked, so it does not depend on having any libraries installed in your workstation (or CI machine).

**Describe alternatives you've considered**

A binary with only a dependency on `glibc` is also possible.

**Additional context**

I can contribute scripts to create this binary, see [this draft](https://github.com/microsoft/vcpkg-tool/compare/main...coryan:feat-automatically-create-linux-executable?expand=1) for a start. I would need some guidance from the vcpkg team to integrate any automation into your release scripts.

**Comment by NancyLi1013** [comment]
Hi @coryan 

Thanks for posting this issue. 
Since you have created a [draft ](https://github.com/microsoft/vcpkg-tool/compare/main...coryan:feat-automatically-create-linux-executable?expand=1)pull request for this feature. You can submit it to `vcpkg-tool `repository. Our colleagues will help review it.
